packages/boost/rollup.config.mjs

```diff
@@ -40,7 +40,6 @@ export default {
     },
   ],
   plugins: [
-    polyfillNode(),
```

Comment:
We're trying to move over to Vite for our build now (sorry for the whiplash) but why remove this for this PR?

Reply:
There was some weird build config (I think it was after importing axios into the boost plugin)

At some point, I took at look at the Neynar plugin and it didn't have the `polyfillNode()`. So I removed it here too and it worked